language/test/Language_test.rb

```diff
@@ -46,6 +46,8 @@
     allow(Y2Country).to receive(:read_locale_conf).and_return(nil)
     allow(subject).to receive(:languages_map).and_return(languages_map)
     allow(subject).to receive(:GetLanguagesMap).and_return(languages_map)
+
+    Yast::Language.main
```

Comment:
this change will depends on fact that main will reset all values. Is it really true?

Reply:
I think so, but it is possible that I overlooked something. Let me recheck it.

Reply:
If I didn't check it wrong, it seems that all the available instances variables in the module are being properly initialized in the `#main` . Namely, 

```ruby
@available_lang_filenames
@cjk_languages
@default_language
@english_names
@ExpertSettingsChanged
@lang2keyboard
@lang2timezone
@language
@language_on_entry
@languages
@languages_directory
@languages_map
@languages_on_entry
@linuxrc_language_set
@localed_conf
@locales
@name
@preselected
@readonly [1]
@reset_recommended
@secondary_items
@selection_skipped
@text_mode
@translation_status
@use_utf8
```

[1] which I'll fix in next commit.